nowcasting_api/tests/test_gsp.py

```diff
@@ -2,12 +2,14 @@
 
```

Comment:
could you remove async from the tests, aprt from the ones that need it?

Reply:
The only test functions requiring asynchronous are those that were facing coroutine errors.